src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,44 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER = 10;
+    private Object[] keys;
+    private Object[] values;
+    private int size;
+    private int index;
+
+    public StorageImpl() {
+        keys = new Object[MAX_NUMBER];
+        values = new Object[MAX_NUMBER];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            if (key == keys[i] || (keys[i] != null && keys[i].equals(key))) {
+                values[i] = value;
+                return;
+            }
+        }
+        keys[index] = key;
+        values[index] = value;
+        size++;
+        index++;
     }
 
     @Override
     public V get(K key) {
+        for (int i = 0; i < size; i++) {
+            if ((keys[i] != null && keys[i].equals(key))
+                    || key == keys[i]) {
```

Comment:
```suggestion
            if (key == keys[i] || (keys[i] != null && keys[i].equals(key))) {
```
That makes more sense.